### DISCUSSION TITLE
v0.10.1

### DISCUSSION THREAD

**Discussion by Blinue** [Blinue/Magpie#537] [Announcements]
1. 修复了启用“始终以管理员身份运行”后无法启动的问题 #511
2. 优化了对 RPG Maker MZ 的支持 #508
3. 现在无需以管理员身份运行也可以为管理员身份的窗口创建配置文件
4. 支持繁体中文 #518（感谢 @NeilTohno）
5. 修复了缩放时无法和高权限窗口交互的问题 #515
6. 修复了 RAVU_Zoom_R3 的错误 #516 

<hr /><em>This discussion was created from the release <a href='https://github.com/Blinue/Magpie/releases/tag/'>v0.10.1</a>.</em>

**Comment by JohnnyJosda** [comment]
有没有兄弟试过magpie内置的两种锯齿(FXAA SMAA)效果怎么说，我玩的游戏用的MSAA代价比较大，想试一下magpie的抗锯齿

**Reply by hooke007** [reply]
不管怎样只有升格渲染然后采样做抗锯齿的效果才会好（游戏内的实现方式）。
仅凭捕获的截图做后期AA必然只会更糊，你可以AA前再用高质量的效果放大一次（作为对抗质量损失的一种workaround

我一般用降采样产生的"自然AA"，多叠几个x2最后bicubic回来，性能代价也不会低（不过我玩的老游戏本来就不支持AA）

**Reply by JohnnyJosda** [reply]
细说，怎么操作，没明白

**Reply by hooke007** [reply]
...比如放大一次用一个nnedi3，多叠一个那就再加一个nnedi3
说实话不要抱什么预期，再怎样都比不上游戏的原生AA

**Reply by JohnnyJosda** [reply]
👌

**Reply by Blinue** [reply]
只想抗锯齿的话试试 reshade

**Reply by JohnnyJosda** [reply]
我是打算使用fsr+smaa这样

**Reply by khrisround2** [reply]
hey so, maybe, (serious potential if actual colab with reshade) but would integration, However possible, of a dx11/12api, hook for z buffer data like reshade used for "add-on support" (crosire's generic) to maybe then be able to add any New upscaling/sampling algorithms you couldn't have otherwise or at the very least Improve quality/efficiency of any current.. I know it's not possible to do FSR2 because of temporal jargon but..

**Reply by Blinue** [reply]
Unlike reshade, Magpie does not inject into the game process, so it is impossible to get the z-buffer. Magpie aims to be widely usable, not just for 3D games.

**Reply by khrisround2** [reply]
I know I know, it ultimately was a hypothetical (Fragpie, haha). Anyway while I have your attention, is there a way, and maybe i'm just dumb or haven't dug deep enough but, to passthrough HDR (games) through?

**Reply by Blinue** [reply]
If HDR is enabled, you should use the game’s native full screen mode for the best experience. There are no plans to support HDR at the moment.

**Comment by JohnnyJosda** [comment]


**Reply by hooke007** [reply]
锐化只会加重锯齿。。。

**Comment by hacpark** [comment]
The shortcut requirement is too strict. I've used END+HOME for shortcut until v0.9.1, but it is blocked. I think you can make more options.

**Reply by Blinue** [reply]
The hotkey function has been rewritten and is now more reliable, and can detect hotkey conflicts. However, the trade-off is that it no longer supports non-standard key combinations.

**Reply by hacpark** [reply]
I see. Then, no plan to support legacy mode in the future?

**Reply by Blinue** [reply]
Non-standard keyboard shortcuts are no longer supported. Nevertheless, thank you for your feedback.

**Comment by Zinoujoker** [comment]
hi 👋

**Comment by NeilTohno** [comment]
还是建议一下更新版本，先把语言更新了，
我那个用繁化姬搞的繁体不够看的

**Reply by Blinue** [reply]
快了

**Comment by MinaseInorin** [comment]
不清楚是不是软件的问题
缩放玩天使騒々的时候出现Direct3D描画报错，场景都是会动的SD画和过场画，按说的更新了Direct3D也没啥用
禁用了Direct3D描画报错几率就小了很多，偶尔还是会报错，而且帧率掉的很厉害

**Reply by Blinue** [reply]
如果游戏报错极有可能是游戏本身的问题，Magpie 不对游戏做修改。你有没有在游戏里设置帧率限制？

**Reply by hooke007** [reply]
你也可以试试obs窗口捕捉游戏录屏，捕捉原理差不多，如果一样报错那肯定不是我们的锅（

**Reply by MinaseInorin** [reply]
> 如果游戏报错极有可能是游戏本身的问题，Magpie 不对游戏做修改。你有没有在游戏里设置帧率限制？

帧率都是设置无限制，刚刚测试改帧率一样报错，最后禁用Direct3D描画基本解决了，就是有掉帧的错觉稍微卡卡的

**Reply by MinaseInorin** [reply]
> 你也可以试试obs窗口捕捉游戏录屏，捕捉原理差不多，如果一样报错那肯定不是我们的锅（

怎么设置啊

**Reply by Blinue** [reply]
是在进入全屏时报错吗？

**Reply by MinaseInorin** [reply]
> 是在进入全屏时报错吗？

对的

**Reply by Blinue** [reply]
我会调查，方便的话请提一个 issue

**Reply by MinaseInorin** [reply]
> 我会调查，方便的话请提一个 issue
提了
https://github.com/Blinue/Magpie/issues/588#issue-1698547477

**Reply by kato-megumi** [reply]
I also have that problem with same game. Disable "Simulate exclusive fullscreen when scaling" might solve that problem.

**Comment by animeojisan** [comment]
Magpie is a great app. Thank you for creating it.

In Windows 11 environment

I use it to watch videos on Youtube, Netflix, etc.

In v0.9.1 & v0.10.0-preview1, by selecting "Graphics Capture" for "Capture Method", the upscaling filter could be used even in full screen mode.

Filters such as Anime4K_Restore worked well and produced sharp images.

I can't do it after v0.10.0-preview2, and v0.10.1 can't do it either.

The ability to use filters in full screen mode was very useful.

It would be helpful if you could make it available as in the previous version.

**Reply by Blinue** [reply]
The original intention of prohibiting scaling maximized and full-screen windows was to prevent misoperation. I will provide an option to restore the previous behavior.

**Reply by animeojisan** [reply]
That is very good information‼　 I look forward to seeing it implemented.

I appreciate it.

**Comment by ChaoseaterzzZ** [comment]
我台式电脑用最新版本放大器，又出现2个鼠标的问题了

**Reply by ChaoseaterzzZ** [reply]
不是，我弄错了，是我远程控制的问题，一个是被控电脑鼠标，一个是我远程控制设备鼠标，远程控制设备鼠标也变成游戏内光标样式了，不是放大器的问题。
